google/cloud/storage/benchmarks/throughput_options.h

```diff
@@ -51,6 +51,9 @@ struct ThroughputOptions {
   };
   std::vector<bool> enabled_crc32c = {false, true};
   std::vector<bool> enabled_md5 = {false, true};
+  std::string rest_endpoint = "https://storage.googleapis.com";
+  std::string grpc_endpoint = "storage.googleapis.com";
+  std::string direct_path_endpoint = "google-c2p:///storage.googleapis.com";
```

Comment:
Just pointing out that this changed from `xds-c2p` to `google-c2p`

Reply:
Ditto.